scipy/optimize/_qap.py

```diff
@@ -0,0 +1,389 @@
+import numpy as np
+import math
+from . import linear_sum_assignment, minimize_scalar, OptimizeResult
+
+
+def quadratic_assignment(
+    cost_matrix,
+    dist_matrix,
+    seed_cost=[],
+    seed_dist=[],
+    maximize=False,
+    n_init=1,
+    init_method="barycenter",
+    max_iter=30,
+    shuffle_input=True,
+    eps=0.05,
+):
+    """
+    Solve the quadratic assignment problem
+
+    This class solves the Quadratic Assignment Problem and the Graph
+    Matching Problem (QAP) through an implementation of the Fast
+    Approximate QAP Algorithm (FAQ) (these two problems are the same up
+    to a sign change) [1].
```

Comment:
```suggestion
    to a sign change) [1]_.
```
Here and below. See https://numpy.org/devdocs/docs/howto_document.html#howto-document #14